akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -6,10 +6,9 @@ package akka.cluster.ddata
 
 import scala.annotation.tailrec
 import scala.collection.immutable
-
 import akka.cluster.Cluster
 import akka.cluster.UniqueAddress
-import akka.util.HashCode
+import akka.util.{ unused, HashCode }
```

Comment:
Not this PR's fault perhaps but why is the annotation in `akka.util` and not in `akka.annotation`?